### PR TITLE
Use github actions

### PR DESCRIPTION
**Pull request by nsurbay**
- Build and test linux package
- Build android package on dev-next and master
- Build PYQBDI linux on dev-next and master

### CODE REVIEW

File: .github/workflows/android.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        run: |
+            python3 build.py prepare ${{ env.QBDI_PLATFORM }}-${{ matrix.QBDI_ARCH }}
+            python3 build.py build ${{ env.QBDI_PLATFORM }}-${{ matrix.QBDI_ARCH }}
+            python3 build.py package ${{ env.QBDI_PLATFORM }}-${{ matrix.QBDI_ARCH }}
+            python3 build.py clean ${{ env.QBDI_PLATFORM }}-${{ matrix.QBDI_ARCH }}
+      - name: Build Package
+        run: |
+          mkdir build
+          cd build
+          cmake -DQBDI_PLATFORM=${{ env.QBDI_PLATFORM }} \
```
Comment (romainthomas) [inline .github/workflows/android.yml:67]: Could be nice to use ninja

Comment (nsurbay) [reply .github/workflows/android.yml:67]: only for android, or for linux too ?

Comment (romainthomas) [reply .github/workflows/android.yml:67]: Both

**Comment by romainthomas** [comment]
Ok for me

**Pull request merged**